wwwroot/assets/scripts/countries.js

```diff
@@ -0,0 +1,47 @@
+export default function () {
```

Comment:
How about if you have as entity: Country then you just create a list of countries.

Reply:
I am not sure I follow you. The countries object is already a list of country entities.